middle_end/flambda2/to_cmm/to_cmm_static.ml

```diff
@@ -44,8 +44,8 @@ let rec static_block_updates symb env res acc i = function
       static_block_updates symb env res acc (i + 1) r
     | Dynamically_computed (var, dbg) ->
       let env, res, acc =
-        C.make_update env res dbg Word_val ~symbol:(C.symbol ~dbg symb) var
-          ~index:i ~prev_updates:acc
+        C.make_update env res dbg C.Update_kind.pointers
```

Comment:
If `var` is a tagged immediate this is overly conservative, but it doesn't look like we can check that here. (This preserves the existing behavior.)